environment.go

```diff
@@ -14,5 +14,5 @@ type Environment struct {
 	// the ID of the distribution the package was discovered on
 	DistributionID string `json:"distribution_id"`
 	// the ID of the repository where this package was downloaded from (currently not used)
-	RepositoryID string `json:"repository_id"`
+	RepositoryIDs []string `json:"repository_ids"`
```

Comment:
@Allda what's the reasoning behind making this an array?

An environment is a way to differentiate finding two of the same packages on the same host but in different file system locations. 

Assuming a package must have a single place of origin, I would assume a package could only have one Repository. Is this not the case?

Reply:
If multiple repositories (CPEs) provide a the same package which is currently installed we can't assigned it to only one of them. That's the reason why repositories is an array.